modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,694 @@
+    },
+  ],
+  toolbar: 'mergetags',
+});
+----
+
+In this example, typing `{{string}}` followed by a word boundary character (such as a space, full-stop, question mark or comma) will result in the _string_ being turned into an active merge tag.
+
+==== Merge tags cannot exceed 255 characters in length
+
+In {productname} 6.3 a new character limit was introduced: merge tags cannot, now, exceed 255 characters.
+
+With this change, all merge tags are now subject to this limit including those predefined in the `mergetags_list` option.
+
+In {productname} 6.3, to support this feature, the editor will check all merge tag lengths to ensure they do not exceed 255 characters. 
+
+In the event the merge tag length exceeds this, the item will be inserted as plain text.
+
+NOTE: Some non-ASCII characters or Unicode codepoints count as more than one character.
+
+For information on the **Merge Tags** plugin, see: xref:mergetags.adoc[Merge Tags].
+
+
+=== Spellchecker Pro 3.1.1
+
+The {productname} 6.3 release includes an accompanying release of the **Spellchecker Pro** premium plugin.
+
+**Spellchecker Pro** 3.1.1 includes one fix:
+
+==== Spellchecker Pro could search outside of the {productname} editor for a language match
+
+For spelling, a word’s language was identified by finding the nearest `lang` attribute anywhere in an ancestor HTML element to the word (ie, any elements that contain the word). 
+
+When {productname} was running in Inline mode, that ancestry included HTML ancestors extant outside the {productname} editor instance.
+
+If a `lang` attribute existed anywhere in that ancestry, this ancestral `lang` attribute value overrode all attempts by the user to set the spell checking language for the document.
+
+As of {productname} 6.3, the `lang` attribute inherits the default settings for the {productname} editor body. As a consequence, the editor no longer searches elements external to the editor body when searching for `lang` attributes.
+
+For information on the **Spellchecker Pro** plugin, see: xref:introduction-to-tiny-spellchecker.adoc[Spellchecker Pro].
+
+=== Spelling Autocorrect 1.0.1
+
+The {productname} 6.3 release includes an accompanying release of the **Spelling Autocorrect** premium plugin.
+
+**Spelling Autocorrect** 1.0.1 includes one fix:
+
+==== Fixed format changes at the start of sentences when auto-capitalization was on
+
+When **Spelling Autocorrect** was active and had Capitalization enabled (ie had `autocorrect_capitalize: true` set), any inline formatting applied at the beginning of a new sentence, after a full stop (period) but before the start of the word, would be lost when the word was auto-capitalized. Instead, the format at the end of the previous sentence was incorrectly applied. 
+
+This update to the **Spelling Autocorrect** premium plugin removed redundant ‘pre-text’ punctuation, before the word at the start of the sentence being capitalized, from start and replacement of text-pattern.
```

Comment:
This line shouldn't be documented, the text-patterns used in the spelling autocorrect plugin are all internal and private.

Reply:
To quote my commit message:

> Went generic on explanation of the change to the Spelling Autocorrect plugin.